src/EoS/Nuclear/CPU_EoS_Nuclear.cpp

```diff
@@ -217,8 +238,15 @@ static real EoS_DensEint2Pres_Nuclear( const real Dens_Code, const real Eint_Cod
    real Temp_MeV  = NULL_REAL;
    real Entr      = NULL_REAL;
    real Pres_CGS  = NULL_REAL;
+   real mu_nu     = NULL_REAL;
```

Comment:
`mu_nu` is not dimensionless. Maybe it is better to specify the unit in the name.